fetch.js

```diff
@@ -167,6 +167,11 @@
 
     this._initBody = function(body) {
       this._bodyInit = body
+
+      delete this._bodyText
+      delete this._bodyBlob
+      delete this._bodyFormData
```

Comment:
That's where I had them initially. I moved them to initBody since it's responsible for setting them. For future maintenance it seemed easier if they are kept close. What do you think is best?